compose/env/db.env

```diff
@@ -0,0 +1,11 @@
+# File required to configure and run integration tests in local env.
```

Comment:
Unfortunately, it needs to be named like this for the integration tests to get setup. It is being called by the php setup tests scripts, and otherwise, the setup will fail. I'll look further into this.

Reply:
Ah, I see. Can we put it in a `env/tests` directory?

Reply:
Oof. That sounds messy, haha. Probably easier to keep it as it is.